[2025-03-30T02:39:03.091+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-03-30T02:39:03.105+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: ecommerce_data_pipeline.fetch_data scheduled__2025-03-30T02:38:00+00:00 [queued]>
[2025-03-30T02:39:03.112+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: ecommerce_data_pipeline.fetch_data scheduled__2025-03-30T02:38:00+00:00 [queued]>
[2025-03-30T02:39:03.113+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 2
[2025-03-30T02:39:03.126+0000] {taskinstance.py:2890} INFO - Executing <Task(PythonOperator): fetch_data> on 2025-03-30 02:38:00+00:00
[2025-03-30T02:39:03.134+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=4777) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-03-30T02:39:03.136+0000] {standard_task_runner.py:72} INFO - Started process 4779 to run task
[2025-03-30T02:39:03.137+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'ecommerce_data_pipeline', 'fetch_data', 'scheduled__2025-03-30T02:38:00+00:00', '--job-id', '569', '--raw', '--subdir', 'DAGS_FOLDER/Gestioncommands.py', '--cfg-path', '/tmp/tmp_rnuqeyx']
[2025-03-30T02:39:03.138+0000] {standard_task_runner.py:105} INFO - Job 569: Subtask fetch_data
[2025-03-30T02:39:03.183+0000] {task_command.py:467} INFO - Running <TaskInstance: ecommerce_data_pipeline.fetch_data scheduled__2025-03-30T02:38:00+00:00 [running]> on host 778b271a0247
[2025-03-30T02:39:03.261+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='admin' AIRFLOW_CTX_DAG_ID='ecommerce_data_pipeline' AIRFLOW_CTX_TASK_ID='fetch_data' AIRFLOW_CTX_EXECUTION_DATE='2025-03-30T02:38:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-03-30T02:38:00+00:00'
[2025-03-30T02:39:03.263+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2025-03-30T02:39:03.263+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2025-03-30T02:39:03.264+0000] {logging_mixin.py:190} INFO - Current task name:fetch_data state:running start_date:2025-03-30 02:39:03.106590+00:00
[2025-03-30T02:39:03.265+0000] {logging_mixin.py:190} INFO - Dag name:ecommerce_data_pipeline and current dag run status:running
[2025-03-30T02:39:03.265+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-03-30T02:39:03.266+0000] {Gestioncommands.py:37} INFO - Fetching data from API...
[2025-03-30T02:39:03.891+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/urllib3/connectionpool.py:1097: InsecureRequestWarning: Unverified HTTPS request is being made to host 'my.api.mockaroo.com'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#tls-warnings
  warnings.warn(

[2025-03-30T02:39:04.635+0000] {Gestioncommands.py:41} INFO - Data fetched successfully
[2025-03-30T02:39:04.637+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-03-30T02:39:04.650+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-03-30T02:39:04.651+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=ecommerce_data_pipeline, task_id=fetch_data, run_id=scheduled__2025-03-30T02:38:00+00:00, execution_date=20250330T023800, start_date=20250330T023903, end_date=20250330T023904
[2025-03-30T02:39:04.679+0000] {logging_mixin.py:190} INFO - Task instance in success state
[2025-03-30T02:39:04.680+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2025-03-30T02:39:04.681+0000] {logging_mixin.py:190} INFO - Dag name:ecommerce_data_pipeline queued_at:2025-03-30 02:39:01.050417+00:00
[2025-03-30T02:39:04.682+0000] {logging_mixin.py:190} INFO - Task hostname:778b271a0247 operator:PythonOperator
[2025-03-30T02:39:04.717+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-03-30T02:39:04.750+0000] {taskinstance.py:3901} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-03-30T02:39:04.752+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
